version.props

```diff
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup>
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
-    <VersionPrefix>3.1.1</VersionPrefix>
+    <VersionPrefix>3.2.0</VersionPrefix>
```

Comment:
Can you update `appveyor.yml` as well please?

https://github.com/justeat/JustEat.StatsD/blob/a6a1319586a6545e21e14ffbed466a22ac350620/appveyor.yml#L2